package.json

```diff
@@ -2,56 +2,73 @@
   "name": "evaluation-request-interception",
   "version": "1.0.0",
   "description": "Idea Pool TA tool evaluation project - Request interception for UI testing",
-  "author": "Various Authors",
+  "author": "Bence Bányász, Gábor Márton, Sándor Simon, Tamás Szűcs",
```

Comment:
Please take a look at this: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors

Reply:
a practical example: https://github.com/gherking/gherking/blob/master/package.json#L33